httpclient5-cache/src/main/java/org/apache/hc/client5/http/impl/cache/CacheRevalidatorBase.java

```diff
@@ -131,7 +131,7 @@ void scheduleRevalidation(final String cacheKey, final Runnable command) {
                     scheduledExecutor.schedule(command, executionTime);
                     pendingRequest.add(cacheKey);
                 } catch (final RejectedExecutionException ex) {
-                    log.debug("Revalidation of cache entry with key " + cacheKey + "could not be scheduled: " + ex);
+                    log.debug("Revalidation of cache entry with key {}could not be scheduled: {}", cacheKey, ex);
```

Comment:
You can remove `: {}`, exception will be handled automatically

Reply:
This will be a slight behavior change, but I agree that it's better to record a full stack trace. Needs a space between the first placeholder and the next word `could`.